### DISCUSSION TITLE
zh_TW language pack is done, based on 1.8.1

### DISCUSSION THREAD

**Discussion by alexclassroom** [YOURLS/YOURLS#2944] [Show and tell]
Hi, I am Alex Lion, one of WordPress zh_TW GTE.
I am a software localization engineer and community translator, now I fully localized YOURLS 1.8.1 to zh_TW.

YOURLS zh_TW GitHub repository: [Here](https://github.com/alexclassroom/YOURLS-zh_TW)

I will maintain this language pack for a long time.

If possible, please list this language pack URL to your official list, thanks.

**Comment by dgw** [comment]
Thank you for letting us know! I have proposed that we replace the current zh_TW link (last updated for 1.7) in YOURLS/awesome-yourls#92
